File: Sort_number_ascending.js
function main() {
  let a, b, c;
  [a, b, c] = readline().split(' ').map(Number);
  let arr = [a, b, c];
  arr.sort((x, y) => x - y);
  console.log(arr[0] + " " + arr[1] + " " + arr[2]);
  return 0;
}

File: Sort_number_ascending.cpp
#include <algorithm>
#include <iostream>

using namespace std;

int main() {
  int a, b, c;
  cin >> a >> b >> c;
  int arr[] = {a, b, c};
  sort(arr, arr + 3);

  cout << arr[0] << " " << arr[1] << " " << arr[2] << endl;
  return 0;
}